scripts/bert/finetune_squad.py

```diff
@@ -215,6 +225,26 @@
 
 log.info(args)
 
+if args.comm_backend == 'horovod':
+    import horovod.mxnet as hvd
+    hvd.init()
+    rank = hvd.rank()
+    size = hvd.size()
+    local_rank = hvd.local_rank()
+else:
+    rank = 0
+    size = 1
+    local_rank = 0
+
+if args.dtype == 'float16':
+    # patch AMP due to issue: https://github.com/apache/incubator-mxnet/issues/17409
+    from mxnet.contrib import amp
+    ops = ['_contrib_interleaved_matmul_encdec_qk', '_contrib_interleaved_matmul_encdec_valatt',
+           '_contrib_interleaved_matmul_selfatt_qk', '_contrib_interleaved_matmul_selfatt_valatt']
+    amp.lists.symbol.WIDEST_TYPE_CASTS.extend(ops)
```

Comment:
What is this?

Reply:
A bug in AMP. But the fix was merged to mxnet nightly, so I'll just remove these temporary fixes.

Reply:
Backported to 1.6 release?

Reply:
yes